test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -19,10 +19,10 @@
       podTemplatePath: config/pod-with-pod-anti-affinity.yaml
   params:
     - numNodes: 500
-      numInitPods: 500
-      numPodsToSchedule: 1000
+      numInitPods: 100
```

Comment:
Let's use the same config as the existing tests for now:

https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_bench_test.go#L48-L50

Reply:
The test `BenchmarkSchedulingPodAntiAffinity` has customized params.

https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_bench_test.go#L76-L77

Reply:
Ack

Reply:
Another problem for test `BenchmarkSchedulingPodAntiAffinity`:

It needs to set different label value for different node
https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_bench_test.go#L87-L91

We need some dynamic value for labels.